Dear/Dear/Cell/PatientPillBagTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  PatientPillBagTableViewCell.swift
+//  Dear
+//
+//  Created by Tempnixk on 2022/07/27.
+//
+
+import UIKit
+
+class PatientPillBagTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var cellStackView: UIStackView!
+    
+    @IBOutlet weak var month: UILabel!
+    
+    @IBOutlet weak var date: UILabel!
+    
+    @IBOutlet weak var day: UILabel!
+    
+    @IBOutlet weak var title: UILabel!
+    
+    @IBOutlet weak var toNurse: UILabel!
+    
+    @IBOutlet weak var moreButton: UIButton!
+    
+    @IBOutlet weak var borderView: UIView!
```

Comment:
코드가 너무 길어져서 붙이는게 어떨가요?

```suggestion
    @IBOutlet weak var cellStackView: UIStackView!
    @IBOutlet weak var month: UILabel!
    @IBOutlet weak var date: UILabel!
    @IBOutlet weak var day: UILabel!
    @IBOutlet weak var title: UILabel!
    @IBOutlet weak var toNurse: UILabel!
    @IBOutlet weak var moreButton: UIButton!
    @IBOutlet weak var borderView: UIView!
```